app/scripts/lib/constants.js

```diff
@@ -113,5 +113,18 @@ module.exports = {
   DOWNLOAD_LINK_TEMPLATE_IOS: 'https://app.adjust.com/2uo1qc?campaign=%(campaign)s&creative=%(creative)s&adgroup=ios&fallback=https://itunes.apple.com/app/apple-store/id989804926?pt=373246&ct=adjust_tracker&mt=8', //eslint-disable-line max-len
 
   MOZ_ORG_SYNC_GET_STARTED_LINK: 'https://mozilla.org/firefox/sync?utm_source=fx-website&utm_medium=fx-accounts&utm_campaign=fx-signup&utm_content=fx-sync-get-started', //eslint-disable-line max-len
+
+  // 20 most popular email domains, used for metrics. Matches the list
+  // we use in the auth server, converted to a map for faster lookup.
+  POPULAR_EMAIL_DOMAINS: [
```

Comment:
Any objections to putting this in https://github.com/mozilla/fxa-shared? We could probably migrate auth-server to use it as well.